server/models/Notification.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+  type: {
+    type: String,
+    enum: ['account', 'appointment', 'messages', 'payment'],
+    lowercase: true,
+  },
+  description: {
+    type: String,
+    default: "",
+  },
+  read: {
+    type: Boolean,
+    default: false,
+  },
+  createdBy: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "Profile",
+  },
+  creatorName: {
```

Comment:
This field is redundant, we can instead obtain the creator's name from their respective profile document (I would suggest leaving it in and fixing in another PR to make it easier)

Reply:
Removed.